packages/stellar_solana_bridge/README.md

```diff
@@ -0,0 +1,5 @@
+# Vue 3 + TypeScript + Vite
```

Comment:
please update the README file to reflect the bridge's setup and usage instructions